globus_automate_client/flows_client.py

```diff
@@ -645,7 +643,10 @@ def list_flow_actions(
                 builder.append(f"{field} {value}")
             params["orderby"] = ",".join(builder)
 
-        return self.get(f"/flows/{flow_id}/actions", params=params, **kwargs)
+        self.authorizer = self._get_authorizer_for_flow(flow_id, flow_scope, kwargs)
+        response = self.get(f"/flows/{flow_id}/actions", params=params, **kwargs)
+        self.authorizer = self.flow_management_authorizer
```

Comment:
Is this because the authorizer for the flow is different than the authorizer for managing flows?

Reply:
Yup, the Globus SDK client's can only support a single `authorizer` at a time, and the `authorizer` is used to authenticate the requests (`self.get`). Since we need 1 `authorizer` to handle interacting with the Flows service, and another one for dealing with individual Flows we end up juggling them around like this in the client operations. Setting `self.authorizer` to be `flow_management_authorizer` sets the Flows client up to be used to hit the Flow's service again.